src/test/java/jivox/DeadlineTest.java

```diff
@@ -0,0 +1,44 @@
+package jivox;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.time.LocalDateTime;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import jivox.task.Deadline;
+
+public class DeadlineTest {
+
+    private Deadline deadline;
+    private LocalDateTime deadlineDateTime;
+
+    @BeforeEach
+    public void setUp() {
+        deadlineDateTime = LocalDateTime.of(2023, 2, 15, 13, 30);
+        deadline = new Deadline("Submit report", deadlineDateTime);
+    }
+
+    @Test
+    public void testGetType() {
+        assertEquals("D", deadline.getType());
+    }
+
+    @Test
+    public void testGetDeadline() {
+        assertEquals(deadlineDateTime, deadline.getDeadline());
+    }
+
+    @Test
+    public void testSaveFormat() {
+        String expected = "D | 0 | Submit report | 2023-02-15 13:30";
+        assertEquals(expected, deadline.saveFormat());
+    }
+
+    @Test
+    public void testToString() {
```

Comment:
Do look into the naming conventions for junit test. Testname should express a specific requirement. Here is a [link](https://medium.com/@stefanovskyi/unit-test-naming-conventions-dd9208eadbea) for reference!